common/xmla-as-connectivity.md

```diff
@@ -60,4 +60,91 @@ Provider=MSOLAP;Data Source=servername;Initial Catalog=databasename;Integrated S
\ No newline at end of file
+Most Analysis Services instances support several languages. See [this page for a full list of locale identifiers (LCID)](https://learn.microsoft.com/en-us/sql/relational-databases/system-catalog-views/sys-fulltext-languages-transact-sql?view=sql-server-ver16).
+
+## Fabric/Power BI XMLA Settings
+
+Two admin settings must be switched on to enable the XMLA endpoint in Fabric/Power BI. 
+
+### Enable Tennant XMLA Endpoint 
+
+In the Fabric/Power BI admin portal, the integration setting "Allow XMLA endpoints and Analyze in Excel with on-premises semantic models" must be enabled
+
+At the tenant level, the setting may be restricted to only certain users. If the setting is restricted in your organization, ensure all required users are allowed to use the XMLA endpoint at the tenant level.
+
+![Tennant Admin Setting](~/images/common/XMLASettings/TennantAdminSetting.png)
+
+### Enable XMLA Read Write on Capacity
+
+To use the XMLA endpoint, the workspace that hosts a semantic model must be assigned to a capacity (FSku or Power BI Premium Per User), and the capacity must have XMLA ["Read Write" enabled in the capacity settings.](https://learn.microsoft.com/en-us/power-bi/enterprise/service-premium-connect-tools#enable-xmla-read-write)
+
+![Tennant Admin Setting](~/images/common/XMLASettings/CapacityAdminSetting.png)
+
+Read Write is enabled in the Admin Portal by navigating to 
+1. Capacity Settings
+1. Choosing the type of capacity
+3. Selecting the relevant Capacity
+4. Navigating to Power BI Workloads and scrolling down to find the XMLA Endpoint setting choosing "Read Write"
+
+## Additional Fabric/Power BI settings.
+
+### Disable Package Refresh
+
+If another user, other than the semantic model's owner, needs to edit the model through the XMLA endpoint, the security admin setting in Fabric/Power BI called "Block republish and disable package refresh" must be disabled.
+
+![Tennant Admin Setting](~/images/common/XMLASettings/DisablePackageRefresh.png)
+
+
+## Unsupported model types
+Several types of models do not support XMLA connections, [listed below](https://learn.microsoft.com/en-us/power-bi/enterprise/service-premium-connect-tools#unsupported-semantic-models). 
+ 
+The following semantic models aren't accessible by using the XMLA endpoint. These semantic models won't appear under the workspace in Tabular Editor or any other tool.
+
+- Semantic models based on a live connection to an Azure Analysis Services or SQL Server Analysis Services model.
+- Semantic models based on a live connection to a Power BI semantic model in another workspace. 
+- Semantic models with Push data by using the REST API.
+- Semantic models in My Workspace.
+- Excel workbook semantic models
+
```

Comment:
Maybe we should also mention Default Semantic Models here (such as when creating a Datamart or a Lakehouse)? You can open/connect to such models, but [not make changes](https://learn.microsoft.com/en-us/power-bi/enterprise/service-premium-connect-tools#considerations-and-limitations)

Reply:
Good point, updated accordingly.